### DISCUSSION TITLE
Connecting TE3 to XLMA endpoint

### DISCUSSION THREAD

**Discussion by InsightfulAnalytics** [TabularEditor/TabularEditor3#1306] [🙋♀️TE3 Q&A] [answered]
Looking at the TE3 docs, it is possible to edit a model through the XLMA endpoint.
https://docs.tabulareditor.com/te3/powerbi-xmla.html

I go to --> open model from DB, but all I see is this (I don't have a PBID file currently open)

![image](https://github.com/TabularEditor/TabularEditor3/assets/30970855/099be84b-33a2-4284-b1a9-1b9ce4fa4f68)

As I can see in the guy in cube video, the "load model from DB" window looks like this:

![image](https://github.com/TabularEditor/TabularEditor3/assets/30970855/446c3bd1-b37f-4d29-b426-16327011ecdc)

Not sure why mine is different, is there a setting somewhere I need to change?

I can connect via DAX studio to my XLMA, so i have the tenant setting all correct.

Many thanks,
Tim

**Comment by otykier** [comment, marked as answer]
Hi Tim,
Looks like you’re using the Desktop Edition of TE3. Note that this edition only supports connecting to local instances of Power BI Desktop: https://docs.tabulareditor.com/te3/editions.html

I’m sorry if this wasn’t clear during purchasing. Feel free to reach out to sales@tabulareditor.com if you would like a refund or if you need assistance upgrading to Business Edition (which supports connecting to the XMLA endpoint Power BI PPU workspaces) or Enterprise Edition (which supports any XMLA endpoint).

**Reply by InsightfulAnalytics** [reply]
Hi Daniel, Thanks for the quick response.  That's fine, I've been a license holder for quite a while now I forgot about the different levels of subscription.
Love the work that you guys do, writing DAX wouldn't be the same without TE.

Tim
